tests/integrations/raftstore/test_transfer_leader.rs

```diff
@@ -108,6 +109,46 @@ fn test_pd_transfer_leader<T: Simulator>(cluster: &mut Cluster<T>) {
         assert!(!resp.get_header().has_error(), "{:?}", resp);
         assert_eq!(resp.get_responses()[0].get_get().get_value(), b"v");
     }
+
+    // with multi targets
+    for id in 1..4 {
+        // select multi target leaders to transfer
+        // set `peer` 0 to make sure the new leader must come from `peers`
+        pd_client.transfer_leader(
```

Comment:
Perhaps we should introduce another function like `transfer_leader_multi_target`